affirm/src/main/java/com/affirm/android/model/Checkout.java

```diff
@@ -94,6 +99,12 @@ public static Builder builder() {
 
         abstract Builder setTotal(Integer value);
 
+        abstract Builder setCurrency(String value);
```

Comment:
Why do you still have the string versions? This would allow the user to enter something other than what we can support

Reply:
Fixed